en/messages.json

```diff
@@ -296,5 +296,23 @@
   "learnWhatToDo": {
     "message": "Learn what to do",
     "description": "Link in banner alert"
+  },
+  "telemetry_screen_header":{
+    "message": "Help us improve the extension",
+    "description": "Header for the Telemetry Opt-In Screen"
+  },
+  "telemetry_toggle_text":{
+    "message": "Share technical and interaction data",
+    "description": "Label for the Telemetry Opt-In toggle"
+  },
+  "telemetry_screen_descr":{
+    "message": "Help us improve features and performance by sharing data such as your hardware configuration and how you use the extension. We never collect data about the sites you visit. Learn more in the $NOTICE$.",
```

Comment:
Where is the text `Mozilla Subscription Services Privacy Notice` coming from? It should be localized, since the document is localized.

Reply:
That will be https://www.mozilla.org/en-US/privacy/subscription-services/ - That is localized afaik

Reply:
Exactly. My question is where the text for the link comes from, right now I have a feeling you would hardcode `Mozilla Subscription Services Privacy Notice`, since there's nothing similar in the strings?

Reply:
Sorry! I am very slow today, ready for the holidays. Added that!